src/graph_notebook/configuration/generate_config.py

```diff
@@ -137,7 +144,9 @@ def generate_default_config():
     parser.add_argument("--config_destination", help="location to put generated config",
                         default=DEFAULT_CONFIG_LOCATION)
     parser.add_argument("--load_from_s3_arn", help="arn of role to use for bulk loader", default='')
-    parser.add_argument("--aws_region", help="aws region your ml cluster is in.", default='us-east-1')
+    parser.add_argument("--aws_region", help="aws region your ml cluster is in.", default=DEFAULT_REGION)
+    parser.add_argument("--proxy_host", help="the proxy host url to route a connection through", default='')
+    parser.add_argument("--proxy_port", help="the proxy port to use when creating proxy connection", default="8182")
```

Comment:
Please also add `args.proxy_host` and `args.proxy_port` to the subsequent `generate_config` call so that the arguments are utilized.